dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,448 @@
+<img src="full_block_subsidy.svg" width="100%" height="279" />
+
+Explanation of terms:
+
+S<sub>full</sub> = The full block subsidy at a given height<br />
+h = The height for which the full subsidy is to be calculated<br />
+P = The coins generated by the first block (1,680,000 X 10<sup>8</sup> on <code>mainnet</code>)<br />
+I = The number of blocks in the subsidy reduction interval (6144 on <code>mainnet</code>)<br />
+c<sub>0</sub> = The base coin subsidy before any reductions (3,119,582,664 on <code>mainnet</code>)<br />
+R<sub>m</sub> = The coin subsidy reduction multiplier (100 on <code>mainnet</code>)<br />
+R<sub>d</sub> = The coin subsidy reduction divisor (101 on <code>mainnet</code>)<br />
+
+===Modified Subsidy Split===
+
+The subsidy split for the full block subsidy MUST be split between Proof-of-Work
+(PoW), Proof-of-Stake (PoS), and the decentralized Treasury as follows:
+
+* PoW - 10%
+* PoS - 80%
+* Treasury - 10% (no change from existing)
+
+Additionally, the resulting PoS subsidy MUST be split evenly between the maximum
+number of votes per block, which, as of this specification, is 5 on all
+networks.
+
+The Treasury subsidy proportion MUST remain the same and otherwise retain all
+existing semantics it has prior to this specification.
+
+The following formulas precisely specify the calculations for the modified
+subsidy splits for arbitrary heights.  This ensures the semantics are fully
+specified for all heights; however, it is important to note that they MUST
+only be applied to <code>mainnet</code>, and current <code>testnet</code>, for
+heights where the new rules are active per the associated on-chain vote.
+
+<!--S_{pow}(h,v) & = & \begin{cases}0 & \text{if } h \le 0,\\S_{full}(1) & \text{if } h = 1,\\\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor & \text{if } 2 \le h < H_{sv},\\0 & \text{if } h \ge H_{sv} \textbf{ and } v < v_{m},\\\left\lfloor\frac{\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor \times v}{v_{b}}\right\rfloor & \text{if } h \ge H_{sv} \textbf{ and } v \ge v_{m},\end{cases}-->
+<!--S_{vote}(h) & = & \begin{cases}0 & \text{if } 0 \le x < H_{sv}-1,\\\left\lfloor\frac{S_{full}(h) \times 8}{10 \times v_{b} }\right\rfloor & \text{if } x \ge H_{sv}-1,\end{cases}-->
+<img src="subsidy_proportions.svg" width="100%" height="464" />
+
+Explanation of terms:
+
+S<sub>full</sub> = The full block subsidy at a given height<br />
+S<sub>pow</sub> = The subsidy for proof of work at a given height for a given number of votes<br />
+S<sub>vote</sub> = The subsidy for a single vote at a given height<br />
+h = The height for which the subsidy is to be calculated<br />
+v = The number of votes included in the block
+H<sub>sv</sub> = The height at which stake validation begins (4096 on <code>mainnet</code>)<br />
+v<sub>b</sub> = The number of votes per block (5 on all networks)
+v<sub>m</sub> = The minimum number of required votes (3 on all networks)
```

Comment:
`<br />` looks to be missing on two of these lines.

Reply:
Added.